sunpy/sun/models.py

```diff
@@ -82,5 +83,120 @@
 _t = {'time': _time, 'luminosity': _tluminosity, 'radius': _tradius,
       'central temperature': _tcentral_temperature}
 evolution = QTable(_t)
-evolution.source = 'Unknown'
+evolution.source = 'Turck-Chieze et al. (1988)'
 evolution.add_index('time')
+
+
+@u.quantity_input
+def differential_rotation(duration: u.s, latitude: u.deg, *, model='howard', frame_time='sidereal'):
+    r"""
+    This function computes the change in longitude over days in degrees.
+
+    Parameters
+    ----------
+    duration : `~astropy.units.Quantity`
+        Number of seconds to rotate over.
+    latitude : `~astropy.units.Quantity`
+        heliographic coordinate latitude in Degrees.
+    model : `str`
+        The differential rotation model to use.
+
+        One of:
+
+        | ``howard`` : Use values from Howard et al. (1990)
+        | ``snodgrass`` : Use values from Snodgrass et. al. (1983)
+        | ``allen`` : Use values from Allen's Astrophysical Quantities, and simpler equation.
+        | ``rigid`` : Use values from `~sunpy.sun.constants.sidereal_rotation_rate`.
+
+    frame_time : `str`
+        One of : ``'sidereal'`` or  ``'synodic'``. Choose 'type of day' time reference frame.
+
+    Returns
+    -------
+    longitude_delta : `~astropy.units.Quantity`
```

Comment:
```suggestion
    `~astropy.units.Quantity`
```
I think is a variable style question for how we want to handle returns.

Reply:
Sure, it's really only for functions that return multiple values that there's a point to naming the returns